hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```diff
@@ -646,12 +646,10 @@ public AbfsRestOperation listPath(final String relativePath, final boolean recur
     abfsUriQueryBuilder.addQuery(QUERY_PARAM_COMP, LIST);
     abfsUriQueryBuilder.addQuery(QUERY_PARAM_INCLUDE, METADATA);
     abfsUriQueryBuilder.addQuery(QUERY_PARAM_PREFIX, getDirectoryQueryParameter(relativePath));
+    abfsUriQueryBuilder.addQuery(QUERY_PARAM_MARKER, continuation);
```

Comment:
if (continuation != null) {
      abfsUriQueryBuilder.addQuery(QUERY_PARAM_MARKER, continuation);
    }

Reply:
This was a redundant check. Same check is already added in AbfsUriQueryBuilder.java
```
public void addQuery(final String name, final String value) {
    if (value != null && !value.isEmpty()) {
      this.parameters.put(name, value);
    }
  }
```